### ISSUE TITLE
Failed to start gauge API: Timed out connecting to 127.0.0.1:53779

### ISSUE THREAD

**Issue by MikeSkilbeck** [getgauge/gauge#1617]
### Describe the bug
Installed gauge for dotnet.
After creating a project and running it the tests fail with error:

Failed to start gauge API: Timed out connecting to 127.0.0.1:53779

Also tried changing the timeout and default port but neither resolved the issue (as per https://stackoverflow.com/questions/56828532/getgauge-failed-to-start-gauge-api-timed-out-connecting-to-127-0-0-1)

### What command(s) did you run when you found the bug?

```
Installed gauge using windows installer selecting dotnet.

Tried with latest version aswell:
gauge install dotnet --file .\gauge-dotnet-0.1.4.zip

Created a new directory and then initialised gauge:
gauge init dotnet

Ran the default feature with no further changes:
gauge run specs

Also tried changing default timeout:
gauge config runner_connection_timeout 40000

Also tried changing GAUGE_PORT and GAUGE_INTERNAL_PORT by adding below environment variables:

GAUGE_PORT= 4000
GAUGE_INTERNAL_PORT = 4000
When I tried this then gauge still listens on a random port but the error is displayed against port 4000, eg Failed to start gauge API: Timed out connecting to 127.0.0.1:4000

```

### Output, stack trace or logs related to the bug 
C:\Users\*myuser*\AppData\Roaming\gauge\plugins\dotnet\0.1.4>dotnet bin\Gauge.Dotnet.dll --start
Microsoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

Listening on port:53789
Error ----------------------------------

[Gauge]
Failed to start gauge API: Timed out connecting to 127.0.0.1:53779

Get Support ----------------------------
        Docs:          https://docs.gauge.org
        Bugs:          https://github.com/getgauge/gauge/issues
        Chat:          https://gitter.im/getgauge/chat

Your Environment Information -----------
        windows, 1.0.6, 2bc49db
        csharp (0.10.6), dotnet (0.1.4), html-report (4.0.8), screenshot (0.0.1), spectacle (0.1.3), xml-report (0.2.2)

#### Gauge (Output of `gauge -v`)

Gauge version: 1.0.6
Commit Hash: 2bc49db

Plugins
-------
csharp (0.10.6)
dotnet (0.1.4)
html-report (4.0.8)
screenshot (0.0.1)
spectacle (0.1.3)
xml-report (0.2.2)

#### Node.js/Java/Python/.Net/Ruby version

dotnet 3.1.201

#### Operating System information

Windows 10 x64 v 1909

#### IDE information

vs code:
Version: 1.44.0 (user setup)

Using gauge extension:
Name: Gauge
Id: getgauge.gauge
Description: Gauge support for VScode.
Version: 0.0.12
Publisher: getgauge
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=getgauge.gauge

**Comment by sriv** [comment]
Apologies for the late response.

Gauge `1.0.6` does not work with `gauge-dotnet` version `0.1.4`. 

Please update gauge to the latest (`1.0.8` at the time of writing this) and try. Alternatively, use `gauge install dotnet` instead of file install. This command checks for the compatibility.

**Comment by MikeSkilbeckPB** [comment]
Issue is resolved with 1.0.8. Many thanks for resolving this.
